configuration.k

```diff
@@ -128,6 +133,10 @@ imports SOLIDITY-SYNTAX
                         <ctType> .Map </ctType>
                         <ctstore> .Map </ctstore>
                         <ctLocation> .Map </ctLocation>
+                        <tempContext> .Map </tempContext>
+                        <tempStore> .Map </tempStore>
+                        <tempdistore> .Map </tempdistore>
+                        <tempmemory> .Map </tempmemory>
```

Comment:
Some cells use a notation like `tempContext`, and others like `tempmemory`. Should this be `tempMemory` instead? We should probably try to stick to a naming convention to avoid confusion.